### ISSUE TITLE
Missing "Date modified" causes dotnet-pack to fail

### ISSUE THREAD

**Issue by kamnel** [autofac/Autofac.AspNetCore.Multitenant#16]
Hi,

I have recently upgraded my projects to .netcore 3.0 and upgraded to Autofac.AspNetCore.Multitenant version 2.0.0. When I am trying to create run `nuget pack` I am getting following error 

```
error : The DateTimeOffset specified cannot be converted into a Zip file timestamp
```

I have isolated the issue to be missing modified date on Autofac.Integration.AspNetCore.Multitenant.dll. 

There was a similar issue reported in another Autofac repository https://github.com/autofac/Autofac.Extensions.DependencyInjection/issues/57 that is now resolved. Appreciate help with fixing the nuget package

Regards
Kamal

**Comment by alsami** [comment]
Hi @kamnel, thanks for reporting this!

@alexmg @tillig we had the same problem [here](https://github.com/autofac/Autofac.Extensions.DependencyInjection/issues/57) for `Autofac.Extensions.DependencyInjection`. Would you mind creating a release again with the same-content to address this issue?

Maybe we should update `Autofac.Extension.DependencyInjection` to `5.0.1` since most people are getting that dependency implicitly by only installing `Autofac.AspNetCore.Multitenant`.

**Comment by alsami** [comment]
I added a PR updating the peer-dependency.

**Comment by alexmg** [comment]
PR has been merged, package updated to `2.0.1`, and release pushed to NuGet.
